Update azure-defender-storage-configure.md (#104447)
MicrosoftDocs/azure-docs: articles/storage/common/azure-defender-storage-configure.md

**Defender for Storage doesn't access the Storage account data, doesn't require you to enable access logs, and has no impact on Storage performance.**

> [!NOTE]

> > Microsoft Defender for Storage customers can now choose to move to a new predictable pricing plan. The pricing model is per-storage account, and high-volume transactions may incur additional overage charges. This new pricing plan will also include all new security features and detections.

Reviewer: ```suggestion
> Microsoft Defender for Storage customers can now choose to move to a new predictable pricing plan. The pricing model is per-storage account, where high-volume transactions may incur additional overage charges. This new pricing plan includes all new security features and detections.
```
